server/php-engine.cpp

```diff
@@ -2027,10 +2026,6 @@ int main_args_handler(int i, const char *long_option) {
       use_utf8();
       return 0;
     }
-    case 2025: {
-      vk::singleton<vk::xgboost::ModelPath>::get().path = optarg;
```

Comment:
Let's leave an option for future experiments

Reply:
done